DataScience/Experimentation.py

```diff
@@ -305,12 +339,13 @@ def detect_namespaces(j_obj, ns_set, marginal_set=None):
     best_learning_rate = results[0].learning_rate
 
     # TODO: Repeat above process of tuning parameters and interactions until convergence / no more improvements.
-    
-    print("Best parameters:")
+
+    elapsed_time = datetime.now() - t0
+    elapsed_time -= timedelta(microseconds=elapsed_time.microseconds)
```

Comment:
What is the purpose of this? Just removing microseconds?

Do:
`print("\nBest parameters found after elapsed time {0}:".format(str(elapsed_time).split('.')[0])`

Reply:
Yes, the microsecond output was too long and also not useful (experimentation takes long enough that second granularity is enough). I'll add a comment to reflect this, unless you have a suggestion for doing this in a more readable way.

Reply:
I'll push a commit with my comment, so you can see if you like it